Test/allocated1/dafny0/FunctionSpecifications.dfy.expect

```diff
@@ -14,7 +14,5 @@ FunctionSpecifications.dfy(137,28): Related location: This is the postcondition
 FunctionSpecifications.dfy(146,2): Error: failure to decrease termination measure
 FunctionSpecifications.dfy(153,2): Error: failure to decrease termination measure
 FunctionSpecifications.dfy(158,2): Error: cannot prove termination; try supplying a decreases clause
-FunctionSpecifications.dfy(164,9): Error: A postcondition might not hold on this return path.
-FunctionSpecifications.dfy(165,14): Related location: This is the postcondition that might not hold.
```

Comment:
How come this error goes away? I thought the effect of this function-height change was that methods get to assume _fewer_ things (in particular, they no longer get to assume axioms from things that previously just happened to appear earlier in the topological order).

If the `A`/`B` methods in `FunctionSpecifications.dfy` were in a separate file, would there be an error reported then? (I hope so. Else, there something that's unsound.)

Reply:
Oh, never mind. I see the `A`/`B` tests were removed. And indeed you also added them in a separate file. All is good.